cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less

```diff
@@ -97,8 +97,20 @@
 			}
 
 			.financial-item + .financial-item {
-				margin-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );;
+				margin-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );
 			}
+
+			dl {
+				dt {
```

Comment:
What's going on with the closing `}` here? Is the indenting just off for the parent `dl`?